model_compression_toolkit/exporter/model_exporter/keras/int8_tflite_exporter.py

```diff
@@ -140,31 +140,32 @@ def _get_pointwise_layer_to_replace_dense(self, wrapped_layer: KerasQuantization
         # Example: Dense kernel with the following shape (3, 20) expects to have input with the
         # next dimensions (BATCH_SIZE, x0, x1, ..., xn, 20).
         # Conv layer expects 4-rank input. Thus, the input is reshaped to (BATCH_SIZE, 1, x0*x1*...*xn, 20)
-        dim = wrapped_layer.layer.input_shape[1:-1]
+        dim = wrapped_layer.input_shape[1:-1]
         target_shape = (1, int(np.prod(dim))) + (dense_kernel.get_shape()[0],)
 
         return Sequential([
             Reshape(target_shape=target_shape),
             wrapped_pw,
-            Reshape(wrapped_layer.layer.output_shape[1:])
+            Reshape(wrapped_layer.output_shape[1:])
         ])
 
     def export(self) -> None:
         """
         Export a fully quantized model to its int8 tflite model.
         """
 
-        def _substitute_model(wrapped_layer: KerasQuantizationWrapper) -> keras.layers.Layer:
+        def _substitute_model(layer_to_substitue: keras.layers.Layer) -> keras.layers.Layer:
```

Comment:
I'm probably missing something, I don't understand why the wrapper has been replaced with a layer.
Also, isn't it too wide condition? Isn't most Keras layers are of type keras.layers.Layer?

Reply:
Before this change, the exporter expected all layers (except for input layers) to be wrapped. After this change, this is not the case as layers that are not quantized, for example, are not wrapped. 
This is not a condition. This is the expected type of input, and all inputs *should* be Keras layers.